dap-python.el

```diff
@@ -233,8 +234,16 @@ strings, for the sake of launch.json feature parity."
        (unless (plist-get conf :cwd)
          (cl-remf conf :cwd))
 
-       (plist-put conf :dap-server-path
-                  (list python-executable "-m" "debugpy.adapter")))
+       (pcase (plist-get conf :request)
+         ("launch"
+          (plist-put conf :dap-server-path
+                     (list python-executable "-m" "debugpy.adapter")))
+         ("attach"
+          ;; TODO: Replace the hardcoded values below with the sub-values of
```

Comment:
Nit: maybe remove `:connect` only in `"attach"`?

Reply:
Done.